ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -317,22 +267,4 @@ open class PillButton: UIButton, TokenizedControlInternal {
             backgroundColor = resolvedBackgroundColor
         }
     }
-
-    private struct Constants {
-        static let bottomInset: CGFloat = 6.0
-        static let horizontalInset: CGFloat = 16.0
-        static let topInset: CGFloat = 6.0
-        static let unreadDotOffset = CGPoint(x: 6.0, y: 3.0)
-        static let unreadDotSize: CGFloat = 6.0
-    }
-
-    private lazy var focusRing: FocusRingView = {
```

Comment:
I think this is sort of my fault for telling you to just stomp on the main files, pretty much, but we need to not delete the focusRing.

Reply:
^ good catch I didn't even notice. Lamine you might be able to benefit from looking at my merge that I did for the tokens branch, it might help